README.md

```diff
@@ -44,6 +44,21 @@ See Helm `values.yaml`, `enterprise` section for configuration examples.
 > Don't have StackStorm Enterprise License?<br>
 > 90-day free trial can be requested at https://stackstorm.com/#product
 
+## Configuration
+
+The default configuration values for this chart are described in `values.yaml`.
+
+## Ingress
+
+Use an ingress if you need to expose HTTP and HTTPS routes from outside the cluster to services
+within the cluster. Ingress provides load balancing, SSL termination and name-based virtual
+hosting. See the ingress section in `values.yaml` for configuration details.
+
+You will need to deploy an ingress controller such as
+https://github.com/helm/charts/tree/master/incubator/aws-alb-ingress-controller or
+https://www.nginx.com/products/nginx/kubernetes-ingress-controller. More are available
+at https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/.
```

Comment:
I think listing so many URLs is excessive. 

Saying that user will need to deploy & configure an Ingress controller of his preference beforehand would be enough. I'm good with including https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/#additional-controllers if you think that can help.